README.md

```diff
@@ -23,6 +23,8 @@ Rustls is a modern TLS library written in Rust.
 <!-- TODO(portable-rustls) CLEANUP & IMPROVE NOTE FOR THIS FORK; REWORK TO AVOID REPEATED INFO -->
 IMPORTANT NOTICE: REGARDLESS OF UPSTREAM `rustls` PROJECT THIS FORK IS NOT CERTIFIED AND NOT PEER-REVIEWED - USE AT YOUR OWN RISK (AS STATED FURTHER BELOW)
 
+XXX TODO MSRV FOR THIS FORK (XXX TODO AS WILL BE DOCUMENTED FURTHER BELOW)
```

Comment:
Can remove from here (and in 2 other places) now that there is MSRV badge near the beginning of this README. MSRV badge may be out of sync between releases but removing this would mean one less thing to keep in sync moving forward.